check-libraries/action.yaml

```diff
@@ -24,19 +32,35 @@ inputs:
     required: true
     description: >
       GitHub token used to post to the pull request that is being checked.
-  comment-on-pr:
+  
+  # Reporting: Labels
+  use-labels:
     required: false
     default: true
+    description: >
+      Whether to use PR labels to show the status of the charm libraries.
+  label-success:
+    required: false
+    default: "Libraries: OK"
+    description: >
+      The label to use when all libraries are in sync
+  label-fail:
+    required: false
+    default: "Libraries: Out of sync"
```

Comment:
Is it possible to set red/green label colors?

Reply:
Once the labels are created, you can style them any way you want. Felt neater to rely on GHs UI than adding a lot of extra options.

Reply:
I mean we could just hardcode red for out of sync, green for in sync?

Reply:
> I mean we could just hardcode red for out of sync, green for in sync?

We can't set it through the `addLabels` function, so that would mean we'd need to add a check for whether the label exists and if it doesn't, create it prior to the fact and set the color. Otherwise, modify the existing label. 

I'll have a look.

Reply:
Fixed